doc/templates/head.php

```diff
@@ -0,0 +1,7 @@
+<head>
+    <title><?php echo $this->page->getTitle(); ?></title>
+    <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="assets/css/bootstrap-theme.min.css" />
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.6/styles/default.min.css">
```

Comment:
If external resources loading is expected then bootstrap could be loaded from CDN too and only host the customizations